### PR TITLE
Ab#5498 add compliance api endpoints

### PR DESCRIPTION
**Pull request by jayson-jensen-pro**

### CODE REVIEW

File: docs/jupiterone-compliance-api.md [hunk 1]
```diff
@@ -0,0 +1,610 @@
...
+**Variables**
+
+- `cursor`: After running the query, a cursor token is returned. This cursor value can be used in the next request to fetch the next page of results.
+- `size`: Optional number to specify the number of frameworks you would like to return
+
+**Query**
+
+```graphql
+query ListStandards($size: Int, $cursor: String) {
+    complianceStandards(size: $size, cursor: $cursor) {
```
Comment (austinkelleher) [inline docs/jupiterone-compliance-api.md:33]: Seems like there are some issues with spacing in each of these

**Review by austinkelleher — CHANGES_REQUESTED** [review]
Thanks for doing this 🙏

File: docs/jupiterone-compliance-api.md (hunk 1 above)
Comment (austinkelleher) [reply docs/jupiterone-compliance-api.md:33]: You can use this for formatting: https://crubier.github.io/graphql-code-pretty-printer/

**Review by hesto2 — APPROVED** [review]
Looks good to me 🎉 I can't verify every field/variable for all of the queries, but at a high level this looks correct to me

File: docs/jupiterone-compliance-api.md (hunk 1 above)
Comment (jayson-jensen-pro) [reply docs/jupiterone-compliance-api.md]: Done, thanks for the resource!

**Review by a-u-h-g — APPROVED** [review]

**Review by austinkelleher — APPROVED** [review]

**Pull request merged**